### PR TITLE
Documentation fix for access_level parameter of gitlab_runner

### PR DESCRIPTION
**Pull request by alinabuzachis**
##### SUMMARY

Documentation description fix for `access_level` parameter of `gitlab_runner`
Fixes: https://github.com/ansible-collections/community.general/issues/2768

##### ISSUE TYPE

- Docs Pull Request

##### COMPONENT NAME

gitlab_runner.py

### CODE REVIEW

**Comment by Akasurde** [comment]
@henkkuli Could you please review this? @alinabuzachis Thanks for putting up the PR.

File: plugins/modules/source_control/gitlab/gitlab_runner.py [hunk 1]
```diff
@@ -77,7 +77,9 @@
     type: bool
   access_level:
     description:
-      - Determines if a runner can pick up jobs from protected branches.
+      - Determines if a runner can pick up jobs only from protected branches.
+      - If set to I(ref_protected), runner can pick up jobs only from protected branches.
+      - If set to I(not_protected), runner can pick up jobs from both protected and unprotected branches.
```
Comment (felixfontein) [inline plugins/modules/source_control/gitlab/gitlab_runner.py:82]:
```suggestion
      - If set to C(ref_protected), runner can pick up jobs only from protected branches.
      - If set to C(not_protected), runner can pick up jobs from both protected and unprotected branches.
```

Comment (alinabuzachis) [reply plugins/modules/source_control/gitlab/gitlab_runner.py]: Thank  you @felixfontein.

**Review by henkkuli — APPROVED** [review]
Looks good to me. The wording now matches my understanding of [the documentation](https://docs.gitlab.com/ee/ci/runners/configure_runners.html#prevent-runners-from-revealing-sensitive-information)(new link):
> You can protect runners so they don’t reveal sensitive information. When a runner is protected, the runner picks jobs created on protected branches or protected tags only, and ignores other jobs.

**Comment by felixfontein** [comment]
@alinabuzachis @Akasurde @henkkuli thanks for improving the docs!

**Pull request merged**